src/Analysis/Ast/Test/GenericsTests.cs

```diff
@@ -642,5 +642,20 @@ class LoggedVar(Generic[T]):
             var analysis = await GetAnalysisAsync(code);
             analysis.Should().HaveVariable("x").OfType(BuiltinTypeId.Int);
         }
+
+        [TestMethod, Priority(0)]
+        public async Task TypeVarNoArguments() {
+            const string code = @"
+from typing import TypeVar
+
+T = TypeVar()
```

Comment:
Should also check other variants like `T = TypeVar(int, float, complex)`, see PEP spec https://www.python.org/dev/peps/pep-0484. Consider `DataTestMethod` with `DataRow` and various code samples.

Reply:
Ok I added some more tests.